packages/storybook/components/source_viewer.test.js

```diff
@@ -0,0 +1,36 @@
+import 'jsdom-global/register';
+import React from 'react';
+import { mount } from 'enzyme';
+import chai, { expect } from 'chai';
+import chaiEnzyme from 'chai-enzyme';
+import SourceViewer from './source_viewer';
+
+chai.use(chaiEnzyme());
+
+describe('<SourceViewer />', () => {
+  let wrapper;
+
+  before(() => {
+    const props = {
+      sources: [
```

Comment:
I'd prefer the following interface:

```typescript
interface Props {
  sources: SourceItem[]
}

interface SourceItem {
  label: string;
  source: string;
}
```

Instead of using tuples.